drag_and_drop_v2/drag_and_drop_v2.py

```diff
@@ -179,6 +190,44 @@ def max_score(self):  # pylint: disable=no-self-use
         """
         return 1
 
+    def get_score(self):
+        """
+        Return the problem's current score as raw values.
+        """
+        if not self._get_raw_earned_if_set():
```

Comment:
Shouldn't this be `if self._get_raw_earned_if_set() is None`?  Otherwise, this will continue to recompute even if self.raw_earned was already computed as 0.